### ISSUE TITLE
The user should choose the font

### ISSUE THREAD

**Issue by oameye** [NonlinearOscillations/HarmonicBalance.jl#49]
The package should let the use choose the font he wants to use. Otherwise it can cause problem when using HarmonicBalance in practice. For example, I use a lot of unicodes in my variable naming. However, the forced `computer modern` font does not recognise the unicodes in ploting such that I get plotting results seen in the figure.
![image](https://user-images.githubusercontent.com/57623933/193592187-e1e41559-6e9a-4c8b-b1e2-3d8e07bdf27e.png)

The solutions would be to not touch the font setting of Plots.jl in the function 

```julia
function _set_Plots_default()
    font = "Computer Modern"
    default(linewidth=2, legend=:outerright)
    default(fontfamily=font, titlefont=font , tickfont=font)
end
```

**Comment by jkosata** [comment]
You can do this via a keyword argument to `plot`
In your case `plot(....; titlefont="Times")` should do.

Any keyword arguments are passed to `Plots.plot`.

**Comment by jkosata** [comment]
This should be doable via a keyword arguments, these are all passed to `Plots.plot`
Here `plot(...; titlefont="sans-serif")

Computer Modern is picked for consistency with the LaTeX labels

**Comment by oameye** [comment]
Ha but I think you misunderstand the problem. You set computer modern by default for every plot. Also the plots outside the package. So if I first plot via the HarmonicBalance package and afterwards plot a random plot like `x = -5:0.1:5; plot(x, x.^2, title = "x₀")`, the plot will be in the "computer modern". I think it is better to not touch the default setting over whole Plots.jl. If you want the HarmonicBalance plots to be in "computer modern" you can do it manually in the `HarmonicBalance.plot` function. In that way, the HarmonicBalance plots are computer modern, but the 'self-made' plots designed by the user during the session are not touched and will be by default in "sans-serif". HarmonicBalance is in that way better designed for further processing data and better compatible with other packges who, for example, plot unicode.

![image](https://user-images.githubusercontent.com/57623933/193613515-eaf3b561-7dfa-4625-aea1-76cdf864b9c9.png)


tldr: I wouldn't touch the 'Plots.default' function and implement that "computer modern" is the default plot font differently 

But it is just a suggestion :p
, one can also just change the default font after it plotted with HarmonicBalance
. I know it is nitpick :stuck_out_tongue: , I can try to implement it differently and make a pull request. ( For clarity, "computer modern" stays default font)
